rustls/src/crypto/cipher.rs

```diff
@@ -214,18 +214,32 @@ impl Nonce {
     /// This is `iv ^ seq` where `seq` is encoded as a 96-bit big-endian integer.
```

Comment:
Anchoring a comment here arbitrarily: You've got a small typo in the commit message header "implment" -> "implement"

Reply:
:+1: I'll keep it in mind for the final squash. Leaving it untouched while the pr settles